Client/src/Utils/Theme/lightTheme.js

```diff
@@ -49,8 +49,8 @@ const lightTheme = createTheme({
     },
     warning: {
       text: "#DC6803",
-      main: "#fdb022",
-      light: "#fffcf5",
+      main: "#ff7a00",
+      light: "#ffb400",
```

Comment:
Should we define new colors for the uptime percentages instead of changing the warning and success colors?

Is the intentio here to change the colors everywhere, or just for the percentages?

@Skorpios604  @danielcj2  I'll defer to you guys for this one so if you are both in agreement here I'm good to go.

Reply:
I don't think `main` needs changing tbh 🤔. But if it is needed for improved contrast then it might be better to define separate colors.

Reply:
Yeah, now that I've had more time to think about it, that's probably the best approach. We're likely going to be using the props in `warning` and `success` for charts in the future, and it'd be better if they remained as they are.

Reply:
Alright makes sense, @Skorpios604 sound good to you?

Reply:
Yup! That sounds good to me. Thanks guys for looking into that for me and for tackling finding the best approach to to go about it.